### ISSUE TITLE
Trim the Gems

### ISSUE THREAD

**Issue by thedannywahl** [instructure/demo_arigato#35]
140 gems in the gemlock..., 50 in the gemfile... probably not using all (most) of those, e.g. SQLite.  Let's cut out the cruft and minimize our surface area...

**Comment by lyonsinbeta** [comment]
Isn't the gemlock generated automatically off the gemfile? I believe it installs related gems as needed which is why the gemlock is often much larger than the gemfile. There may be ways around this though.

**Comment by thedannywahl** [comment]
You're right @lyonsinbeta because I'm a n00b - but still 50 gems has to be excessive.

**Comment by everydaytweed** [comment]
I removed a couple and updated many in the most recent commit. I think it would be helpful to do an audit and see what gems we're actually using and ones we're not.
